projects/packages/jetpack-mu-wpcom/src/features/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-launchpad.php

```diff
@@ -95,14 +111,18 @@ public function can_access() {
 	/**
 	 * Returns Launchpad-related options.
 	 *
+	 * @param WP_REST_Request $request Full data about the request.
+	 *
 	 * @return array Associative array with `site_intent`, `launchpad_screen`,
-	 *               and `launchpad_checklist_tasks_statuses` as `checklist`.
+	 *               `launchpad_checklist_tasks_statuses`, and `checklist`.
```

Comment:
Let's tighten this comment up slightly. It should read `launchpad_checklist_tasks_statuses` as `checklist_statuses`, and `checklist`.

Reply:
This has been done!